snippets/csharp/concepts/linq/how-to-query-a-collection-of-objects_1.cs

```diff
@@ -1,66 +1,62 @@
+            new Student {FirstName = "Debra", LastName = "Garcia", Id = 115, 
                 Year = GradeLevel.ThirdYear, 
-                ExamScores = new List<int>{ 35, 72, 91, 70}},
-            new Student {FirstName = "Hugo", LastName = "Garcia", ID = 118, 
+                ExamScores = new List<int> { 35, 72, 91, 70}},
+            new Student {FirstName = "Hugo", LastName = "Garcia", Id = 118, 
                 Year = GradeLevel.SecondYear, 
-                ExamScores = new List<int>{ 92, 90, 83, 78}},
-            new Student {FirstName = "Sven", LastName = "Mortensen", ID = 113, 
+                ExamScores = new List<int> { 92, 90, 83, 78}},
+            new Student {FirstName = "Sven", LastName = "Mortensen", Id = 113, 
                 Year = GradeLevel.FirstYear, 
-                ExamScores = new List<int>{ 88, 94, 65, 91}},
-            new Student {FirstName = "Claire", LastName = "O'Donnell", ID = 112, 
+                ExamScores = new List<int> { 88, 94, 65, 91}},
+            new Student {FirstName = "Claire", LastName = "O'Donnell", Id = 112, 
                 Year = GradeLevel.FourthYear, 
-                ExamScores = new List<int>{ 75, 84, 91, 39}},
-            new Student {FirstName = "Svetlana", LastName = "Omelchenko", ID = 111, 
+                ExamScores = new List<int> { 75, 84, 91, 39}},
+            new Student {FirstName = "Svetlana", LastName = "Omelchenko", Id = 111, 
                 Year = GradeLevel.SecondYear, 
-                ExamScores = new List<int>{ 97, 92, 81, 60}},
-            new Student {FirstName = "Lance", LastName = "Tucker", ID = 119, 
+                ExamScores = new List<int> { 97, 92, 81, 60}},
+            new Student {FirstName = "Lance", LastName = "Tucker", Id = 119, 
                 Year = GradeLevel.ThirdYear, 
-                ExamScores = new List<int>{ 68, 79, 88, 92}},
-            new Student {FirstName = "Michael", LastName = "Tucker", ID = 122, 
+                ExamScores = new List<int> { 68, 79, 88, 92}},
+            new Student {FirstName = "Michael", LastName = "Tucker", Id = 122, 
                 Year = GradeLevel.FirstYear, 
-                ExamScores = new List<int>{ 94, 92, 91, 91}},
-            new Student {FirstName = "Eugene", LastName = "Zabokritski", ID = 121,
+                ExamScores = new List<int> { 94, 92, 91, 91}},
+            new Student {FirstName = "Eugene", LastName = "Zabokritski", Id = 121,
                 Year = GradeLevel.FourthYear, 
-                ExamScores = new List<int>{ 96, 85, 91, 60}}
+                ExamScores = new List<int> { 96, 85, 91, 60}}
         };
         #endregion
 
-        //Helper method, used in GroupByRange.
+        // Helper method, used in GroupByRange.
         protected static int GetPercentile(Student s)
         {
             double avg = s.ExamScores.Average();
             return avg > 0 ? (int)avg / 10 : 0;
         }
 
-        
-
         public void QueryHighScores(int exam, int score)
```

Comment:
After these changes, this class is more consistent if the `QueryHighScores` method is `static`.

Reply:
```suggestion
        public static void QueryHighScores(int exam, int score)
```